app/javascript/react/components/DayView/DayView.style.tsx

```diff
@@ -0,0 +1,112 @@
+import styled from "styled-components";
+
+import { H6, H5 } from "../Typography";
+import { thresholdsValues } from "../WeekView/WeeklyMockData";
+import { ThresholdsValues } from "../../utils/ThresholdsValues";
+import media from "../../utils/media";
+import * as colors from "../../assets/styles/colors";
+
+interface ValueBarProps {
+  value: number;
+  thresholdsValues: ThresholdsValues;
+}
+
+const COLORS_FOR_RANGES = [
+  { max: thresholdsValues.low, color: colors.green },
+  { max: thresholdsValues.middle, color: colors.yellow },
+  { max: thresholdsValues.high, color: colors.orange },
+  { max: thresholdsValues.max, color: colors.red },
+];
+
+const getColorForValue = (value: number) => {
+  for (let range of COLORS_FOR_RANGES) {
+    if (value <= range.max) {
+      return range.color;
+    }
+  }
+  return colors.grey200;
+};
+
+const calculateBarHeight = (
+  value: number,
+  { max }: ThresholdsValues
+): number => {
+  const definedPercentage = (value / max) * 100;
```

Comment:
did you consider the case when value is a negative number? I know we have temp so it's possible, isn't it? 🤔